ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -317,6 +317,37 @@ def test_paddle_is_floating_point(
     )
 
 
+# sum
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="sum",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), num_arrays=2, shared_dtype=True
+    ),
+)
+def test_paddle_sum(
+    dtypes_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+):
+    input_dtype, x = dtypes_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={"data": x[0]},
```

Comment:
Tests are failing due to missing `backend_to_test` argument.

Reply:
Added the `backend_to_test` argument to the sum test function.

Reply:
Hey, I can't see the `backend_to_test` argument added anywhere, the tests are still failing due to this reason.